CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.2 (2019-04-24)
+
+- Fixed tabset component to avoid throwing ViewDestroyedErrors when inside a modal. [#21](https://github.com/blackbaud/skyux-tabs/pull/21)
```

Comment:
to prevent?

Reply:
I agree that changing "avoid throwing" to "prevent" makes sense. I think we can also tighten up "when inside a modal" to "in modals." ... Also, should "ViewDestroyedErrors" be formatted as a code snippet?

Reply:
Agree on all accounts. Updated!